app/views/layouts/application.html.erb

```diff
@@ -1,11 +1,10 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
     <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
-
-    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag    'application', media: 'all',
+                                              'data-turbolinks-track': 'reload' %>
```

Comment:
sai coding convention:
- trong rubu file thì dùng nháy đôi
- các thành phần tách nhau bởi 1 space
- indent thì chỉ 1 tab = 2 spaces (line 7)

2. tìm hiểu về I18n
sử dụng I18n cho tất cả các text như thế này "Ruby on Rails Tutorial Sample App" trong toàn bộ file html